docs/release-process.md

```diff
@@ -11,109 +11,89 @@
-
-1. Once the final release candidate has rolled out successfully without issue on staging
-   infrastructure, the tagged release shall no longer marked as Pre-Release on the [Github releases](https://github.com/stacks-network/stacks-core/releases/) page. 
-   Announcements will then be shared in the `#stacks-core-devs` channel in the
-   Stacks Discord, as well as the [mailing list](https://groups.google.com/a/stacks.org/g/announce).
-
-1. Finally, the release branch `release/X.Y.Z.A.n` will be PR'ed into the `master` branch, and once merged, a PR for `master->develop` will be opened. 
+The release must be timed so that it does not interfere with a _prepare phase_.
+The timing of the next Stacking cycle can be found [here](https://stx.eco/dao/tools?tool=2).
+A release should happen at least 24 hours before the start of a new cycle, to avoid interfering with the prepare phase.
+
+1. Before creating the release, the _version number_ must be determined, where the factors that determine the version number are discussed in [Versioning](#versioning).
+
+   - First determine whether there are any "non-consensus-breaking changes that require a fresh chainstate".
+      - In other words, the database schema has changed, but an automatic migration was not implemented.
+      - Determine whether this a feature release, as opposed to a hotfix or a patch.
+   - A new branch in the format `release/X.Y.Z.A.n` is created from the base branch `develop`.
+
+2. Enumerate PRs and/or issues that would _block_ the release.
+
+   - A label should be applied to each such issue/PR as `X.Y.Z.A.n-blocker`.
+
+3. Since development is continuing in the `develop` branch, it may be necessary to cherry-pick some commits into the release branch.
+
+   - Create a feature branch from `release/X.Y.Z.A.n`, ex: `feat/X.Y.Z.A.n-pr_number`.
+   - Add cherry-picked commits to the `feat/X.Y.Z.A.n-pr_number` branch
+   - Merge `feat/X.Y.Z.A.n-pr_number` into `release/X.Y.Z.A.n`.
+
+4. Open a PR to update the [CHANGELOG](../CHANGELOG.md) file in the `release/X.Y.Z.A.n` branch.
+
+   - Create a chore branch from `release/X.Y.Z.A.n`, ex: `chore/X.Y.Z.A.n-changelog`.
+   - Add summaries of all Pull Requests to the `Added`, `Changed` and `Fixed` sections.
+      - Pull requests merged into `develop` can be found [here](https://github.com/stacks-network/stacks-core/pulls?q=is%3Apr+is%3Aclosed+base%3Adevelop+sort%3Aupdated-desc).
+        
+        **Note**: GitHub does not allow sorting by _merge time_, so, when sorting by some proxy criterion, some care should be used to understand which PR's were _merged_ after the last release.
+
+5. Once `chore/X.Y.Z.A.n-changelog` has merged, a build may be started by manually triggering the [`CI` workflow](../.github/workflows/ci.yml) against the `release/X.Y.Z.A.n` branch.
+
+6. Once the release candidate has been built and binaries are available, ecosystem participants shall be notified to test the tagged release on various staging infrastructure.
```

Comment:
Are testnets part of the staging infrastructure? They are mentioned below and using both terms might leave the reader wondering which is which.

Reply:
For the line you're referring to about genesis sync where testnet/mainnet are mentioned specifically -  those networks will have public chainstates that any release will need to be compatible with. "staging infrastructure" doesn't always mean it will use a public chainstate, but i felt there was little benefit in adding extra words to describe what is and isn't "staging infra".

Reply:
Got it, I don't think adding the definition adds value either. 

The valuable bit is that the process ensures compatibility with existing public subset of staging infrastructure (testnet and mainnet) and that's why line 80 calls them out specifically. But I don't think we need to explain that here. 

I'll resolve this.